docker-compose.yml

```diff
@@ -1,20 +1,41 @@
----
+version: '3.8'
 services:
-  postgres:
+  db:
     image: postgres:14
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
     environment:
-      POSTGRES_USER: ${DATABASE_USERNAME}
-      POSTGRES_PASSWORD: ${DATABASE_PASSWORD}
-      POSTGRES_DB: pet_rescue_development
+      POSTGRES_DB: myapp_development
+      POSTGRES_USER: ${DATABASE_USERNAME:-postgres}
+      POSTGRES_PASSWORD: ${DATABASE_PASSWORD:-password}
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${DATABASE_USERNAME:-postgres} -d myapp_development"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
 
+  app:
+    build: .
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -b '0.0.0.0'"
+    volumes:
+      - .:/myapp
     ports:
-      - 5432
```

Comment:
Ohh, this needs to stick with the database container -- again, exposure for local console development.